### ISSUE TITLE
[Custom]: Kitty tab bar style

### ISSUE THREAD

**Issue by RAprogramm** [prasanthrangan/hyprdots#943]
### Description

Hello everyone! What do you think about to make default kitty bar style 'Slant' with same colors background and active tab?

![240223_17h48m35s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/70325462/538b679a-838b-475a-950b-688bc69cdc3c)
![240223_17h49m43s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/70325462/a6817fe4-150b-4fcb-b745-b6ff9873fe96)
  

Also the same colors will be waybar modules background and inactive kitty tabs

**Comment by kRHYME7** [comment]
Let me steal this. 😁

**Comment by prasanthrangan** [comment]
The tabs looks great, please feel free to push it ti this repo.

> P.S.: i tried to make theme in 'OneDark' style. But i have some problems with gtk settings for Dophin. If someone would like to help me fix that -> [here is repo](https://github.com/RAprogramm/hyprdots-theme).

Alright, this looks dope! I'll try it and add it to themepatcher list.

**Comment by kRHYME7** [comment]
Can we add a "gallery" page for custom themes? Just to share and collab their themes. 😁

**Comment by prasanthrangan** [comment]
> Can we add a "gallery" page for custom themes? Just to share and collab their themes. 😁

definitely, maybe add a gallery page to Wiki and link all contributors repo/themes in it.

**Comment by its-19818942118** [comment]
Well yeah but for this I think the tab style is kinda basic. Option for more than one . I cant configure kitty it seems super confusing. while I like this tab but I feels super basic. No hard feelings I dont mean to offend. Suggesting more than one tab style

**Comment by RAprogramm** [comment]
I got idea...
Different kitty tabs styles for different themes!
How do you think?

**Comment by its-19818942118** [comment]
Do it

**Comment by its-19818942118** [comment]
Share on discord
